src/base-element.js

```diff
@@ -586,40 +597,78 @@ export class BaseElement {
    * The handler is only invoked by events with trust equal to or greater than
    * `minTrust`. Otherwise, a user error is logged.
    *
-   * @param {string} method
+   * @param {string} alias
    * @param {function(!./service/action-impl.ActionInvocation)} handler
    * @param {ActionTrust} minTrust
    * @public
    */
-  registerAction(method, handler, minTrust = ActionTrust.HIGH) {
+  registerAction(alias, handler, minTrust = ActionTrust.HIGH) {
+    this.registerAction_(
+        alias, handler, minTrust, /* opt_isDefault */ false);
+  }
+
+  /**
+   * Registers the default action for this component.
+   * @param {function(!./service/action-impl.ActionInvocation)} handler
+   * @param {string=} alias
+   * @param {ActionTrust} minTrust
+   */
+  registerDefaultAction(
+    handler,
+    alias = DEFAULT_METHOD,
+    minTrust = ActionTrust.HIGH) {
+    this.registerAction_(
+        alias, handler, minTrust, /* opt_isDefault */ true);
+  }
+
+  /**
+   * @param {string} alias
+   * @param {function(!./service/action-impl.ActionInvocation)} handler
+   * @param {ActionTrust} minTrust
+   * @param {boolean} isDefault Whether this action is the default.
+   * @private
+   */
+  registerAction_(alias, handler, minTrust = ActionTrust.HIGH,
+    isDefault = false) {
     this.initActionMap_();
-    this.actionMap_[method] = {handler, minTrust};
+    this.actionMap_[alias] = {handler, minTrust};
+    if (isDefault) {
+      dev().assert(!this.defaultActionAlias_,
+          'There is already a default action registered.');
+      this.defaultActionAlias_ = alias;
+    }
```

Comment:
Nit: Looks like you can simplify these 3 functions into 2 and remove the `isDefault` param.

```js
registerAction(...) {
}

registerDefaultAction(...) {
  this.registerAction(...);
  dev().assert(!this.defaultActionAlias_, ...);
  this.defaultActionAlias_ = alias;
}
```